### ISSUE TITLE
No English description

### ISSUE THREAD

**Issue by vi** [yangyangwithgnu/use_vim_as_ide#5]
A repository without at least a short English description looks uninviting.

There should be at least the list of IDE features the project brings into Vim.

As far as I see from screenshots, it's 

* Listing of project files;
* Code navigator;
* Smarter code completion;
* Searching in project files;

It is hovewer not clear which languages does the project support, and whether aids for refactoing, build system integration, debugging, testing and deployment are in scope of the project.

Ideally there should be a link to fully translated English README page near the top of default README.

**Comment by yangyangwithgnu** [comment]
A repository without at least a short English description looks uninviting.
-- u know, there are 80+ pages of this project, i have no time to do this huge job. so, if someone wanna translate this project to english, welcome and big thanks, yes, i need yours help.

There should be at least the list of IDE features the project brings into Vim.
-- as you see, the index links on the top of README.md are "the list of IDE features".

As far as I see from screenshots, it's
    Listing of project files;
    Code navigator;
    Smarter code completion;
    Searching in project files;
-- yes, you have a double-hawkeye 

It is hovewer not clear which languages does the project support, and whether aids for refactoing, build system integration, debugging, testing and deployment are in scope of the project.
-- "希望将 vim 打造成 C/C++ 语言的 IDE，而非其他语言", in english, "this project just for C/C++ guys"

Ideally there should be a link to fully translated English README page near the top of default README.
-- this is a goooood ideal! i need help from the world.


-- by the way, how do you get this project? you are chinese? (ง •̀_•́)ง

**Comment by yangyangwithgnu** [comment]
and whether aids for refactoing, build system integration, debugging, testing and deployment are in scope of the project.
-- build system integration in scope of this project; debugging coming soon; refactoing, testing and deployment are out scope of the project

**Comment by vi** [comment]
> by the way, how do you get this project?

This project received many stars, therefore appeared on [trending repositories](https://github.com/trending) Github page, which I monitor.

On this page (and on search page) you only see short description of the repostiory: name, description (currently just `use vim as IDE`) and programming languages. If I see there's something that makes repository less easy to find, get involved or contribute, I often file an issue.

>  You know, there are 80+ pages of this project.

Initially there can be just a little description in English, not the whole 80+ pages.

**Comment by pacuna** [comment]
+1

**Comment by myshov** [comment]
I agree with @vi I found your interesting repository via trending repositories, but unfourtunally I don't know Chinese language to dig into docs.

**Comment by wsdjeg** [comment]
there is no need english !only read VIML

**Comment by xushaobo** [comment]
introduce a plugin for preview yours .md files when you editor in vim.___Plugin 'iamcco/markdown-preview.vim" It is more friendly than the firefox's

**Comment by NormanBB** [comment]
@yangyangwithgnu

What is the progress of the translation project now? Does anyone offers help？

**Comment by MSiam** [comment]
I am also interested in the English version please.
